source/draw/RasterBgr.ooc

```diff
@@ -110,7 +110,7 @@ RasterBgr: class extends RasterPacked {
 						result += (distance) sqrt() / 3
 					}
 				}
-			result /= ((this size width squared() + this size height squared()) as Float sqrt())
+			result /= this size length
```

Comment:
I agree that this one is more then big enough and that I should have avoided it this time. But in general I think we should allow for smaller improvements to the code quality while doing other changes. Else we end up with not getting the other once done.